### PR TITLE
Dj rest

### PR DESCRIPTION
**Pull request by Baranovsky1986**
add api structure for ratu

### CODE REVIEW

File: ratu/serializers.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+
+class RegionSerializer(serializers.Serializer):
+    title = serializers.CharField(max_length=120)
+    description = serializers.CharField()
+    body = serializers.CharField()
```
Comment (Iva-khar) [inline ratu/serializers.py:6]: ???

File: ratu/urls.py [hunk 2]
```diff
@@ -1,7 +1,15 @@
 from django.urls import path
 
 from . import views
-
+from .views import RegionView,DistrictView,CityView,CitydistrictView,StreetView
```
Comment (Iva-khar) [inline ratu/urls.py:4]: Two times` import views`

File: ratu/serializers.py (hunk 1 above)
Comment (Baranovsky1986) [reply ratu/serializers.py:6]: Thank you. i`m understand my mistake

**Comment by criticus** [comment]
this PR has conflicts, please resolve them first

**Pull request closed without merging**